src/background/adblocker.js

```diff
@@ -367,14 +373,7 @@ ${scripts.join('\n\n')}}
   );
 }
 
-async function injectScriptlets(tabId, frameId, url) {
```

Comment:
async removed 

@smalluban wonder if we should not early exit if setup is still pending.